docs/reference/content/guides/indexing.md

```diff
@@ -0,0 +1,181 @@
++++
+title = "Indexing"
+[menu.main]
+  parent = "Reference Guides"
+  weight = 100
+  pre = "<i class='fa fa-file-text-o'></i>"
++++
+
+# Indexing
+
+Morphia provides annotations that allow developers to define indexes for a collection to be defined alongside the other mapping data on 
+an entity's source.  In addition to the familiar ascending/descending index types, Morphia and MongoDB support [TTL]({{< docsref 
+"core/index-ttl/" >}}), [text]({{< docsref "core/index-text/" >}}), and [geospatial]({{< docsref "applications/geospatial-indexes/" >}})
+ indexes.   When defining [text]({{< ref "#text-indexing" >}}) indexes there are certain restrictions which will be covered below.  Full
+  details for all these types are available in the manual.
+ 
+There are two ways to define indexes:  at the class level and at the field level.
+
+## Class Level Indexes
+
+Class level indexing begins with the [`@Indexes`]({{< ref "guides/annotations.md#indexes" >}}) annotation.  This is a container 
+annotation whose sole purpose is to hold a number of [`@Index`]({{< ref "guides/annotations.md#index" >}}) annotations.  This annotation 
+has two primary components to cover here:  `fields` and `options`.  An index definition would take the following form:
+
+```java
+@Entity
+@Indexes({
+    @Index(fields = @Field("field2", type = DESC)),
+    @Index(fields = @Field("field3"))
+})
+public class IndexExample {
+    @Id
+    private ObjectId id;
+    private String field;
+    @Property
+    private String field2;
+    @Property("f3")
+    private String field3;
+}
+```
+
+### Fields
+
+The fields to use in an index definition are defined with the [`@Field`]({{< ref "guides/annotations.md#field" >}}) annotation. An 
+arbitrary number of `@Field`s can be given but at least one must be present.
+
+#### type()
+*Default: IndexType.ASC*
+     
+Indicates the "type" of the index (ascending, descending, geo2D, geo2d sphere, or text) to create on the field.
+  
+*See [IndexType](/javadoc/org/mongodb/morphia/utils/IndexType.html)*
+     
+#### value()
```

Comment:
Move this up above type() as it comes first in the example.  Make it clear that "fields"/"field3" is the value().